tests/pages/firefox/whatsnew/whatsnew_75.py

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage
+
+
+class FirefoxWhatsNew75Page(FirefoxBasePage):
+
+    URL_TEMPLATE = '/{locale}/firefox/75.0/whatsnew/all/{params}'
+
+    _signed_out_sync_button_locator = (By.CSS_SELECTOR, '.show-signed-out .qa-sync-button')
+    _signed_in_monitor_button_locator = (By.CSS_SELECTOR, '.show-signed-in .qa-monitor-button')
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda s: self.seed_url in s.current_url)
+        el = self.find_element(By.TAG_NAME, 'body')
+        self.wait.until(lambda s: 'state-fxa-default' not in el.get_attribute('class'))
```

Comment:
The signed-in test is failing intermittently for me, because it's waiting for this class to be removed from the body before running. This class is never applied however, because it's expecting the page to use `mozilla-fxa.js`. So the test runs early, sometimes before the content switches state.

Changing the page to use the standard fxa JS library should probably make things more reliable. Although if you didn't want to do that for some reason, you could also change the default class to something custom here, and then remove that in your callback.